site/src/components/tabs.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Tabs
+description: Tabs component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - tabs
+---
+
+## Tabs
+
+Tabs component consists of navigating to the previous or next tab and well and clicking on a tab. The result will change the content in the `tab-item-container` to display the current selected item. To do this the `[data-tab-control="1"]` and `[data-tab-item="1"]` must have matching numbers as these act as the index for navigation.
+
+```html
+<tab-container data-tab-container>
+	<div class="tabs">
+		<button type="button" class="tab-previous" title="Previous tab" data-tab-nav="previous">
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448">
+					<path
+						class="fill-current-color"
+						d="M448 224H62.625l148.688 148.688-22.625 22.625L1.375 208 188.688 20.688l22.625 22.625L62.625 192H448v32z"
+					/>
+				</svg>
+			</span>
+		</button>
+		<div class="tab-list" role="tablist" aria-orientation="horizontal">
+			<button
+				class="tab-control"
```

Comment:
Please add `inner-focus` to the tab controls to fix the outline appearance:

![image](https://user-images.githubusercontent.com/57374379/214175225-9d379c9a-8564-47e5-afeb-451eed4d405e.png)

Reply:
are you wanting to hide the focus?

Reply:
updated